client/keys.go

```diff
@@ -173,20 +173,60 @@ func (key *Decrypter) Decrypt(rand io.Reader, msg []byte, opts crypto.DecrypterO
 	}
 
 	if ok {
-		// If opts.SessionKeyLen is set, we must perform a variation of
```

Comment:
shall we keep this comment?

Reply:
I don't think it's particularly helpful.